README.md

```diff
@@ -1,12 +1,22 @@
-## Opportunity Calendar
+## Opportunity Calendar <img src="https://github.com/Samridhi-98/Images/blob/master/Images/op-logo.png" width="3%">
 
+<p>
+
+[![GitHub contributors](https://img.shields.io/github/contributors/adityabisoi/REST-fetch)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/graphs/contributors/)
+[![Issues](https://img.shields.io/github/issues/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/issues)
+[![PRs](https://img.shields.io/github/issues-pr/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend/pulls)
+[![Forks](https://img.shields.io/github/forks/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend)
+[![Stars](https://img.shields.io/github/stars/Girl-Code-It/Opportunity-Calendar-Backend)](https://github.com/Girl-Code-It/Opportunity-Calendar-Backend)
+[![Visits Badge](https://badges.pufler.dev/visits/Girl-Code-It/Opportunity-Calendar-Backend)](https://badges.pufler.dev)
 [![Build Status](https://www.travis-ci.com/Girl-Code-It/Opportunity-Calendar-Backend.svg?branch=develop)](https://www.travis-ci.com/Girl-Code-It/Opportunity-Calendar-Backend)
 
+</p>
+
 Opportunity Calendar is the one-stop place to refer important opportunities available in tech-space like newly posted jobs, internships, hackathons, tech-conferences, scholarships etc.
 
 The frontend of the same project can be found [here](https://github.com/Girl-Code-It/Opportunity-Calendar-Frontend)
 
-## Table of Contents
+## Table of Contents <img src="https://github.com/Samridhi-98/Images/blob/master/Images/rocket.svg" width="3%">
```

Comment:
These images are not needed, and if you want to add them, they should be added in the assets folder of this project and in not your account.

Reply:
okay, I will add them to the assets folder.